### PR TITLE
Add Cinder to ControlPlane CR

### PR DESCRIPTION
**Pull request by dprince**

### CODE REVIEW

**Review by abays — COMMENTED** [review]
Looking good so far

**Comment by abays** [comment]
I think this might be missing changes to `custom-bundle.Dockerfile` to inject the actual Cinder operator into the bundle.  When I build and deploy it I don't see a Cinder controller-manager get created yet.

**Comment by abays** [comment]
`main.go` also needs to register the Cinder `v1beta1` scheme

**Comment by Akrog** [comment]
It doesn't deploy the Cinder operator for me either.

**Comment by dprince** [comment]
Sorry guys, That was a sloppy rebase on me. Re-adding those things and will re-test again myself

File: custom-bundle.Dockerfile [hunk 1]
```diff
@@ -27,6 +27,7 @@ FROM quay.io/openstack-k8s-operators/mariadb-operator-bundle:latest as mariadb-b
 FROM quay.io/openstack-k8s-operators/rabbitmq-cluster-operator-bundle:latest as rabbitmq-bundle
 FROM quay.io/openstack-k8s-operators/placement-operator-bundle:latest as placement-bundle
 FROM quay.io/openstack-k8s-operators/glance-operator-bundle:latest as glance-bundle
+FROM quay.io/openstack-k8s-operators/cinder-operator-bundle:latest as cinder-bundle
```
Comment (abays) [inline custom-bundle.Dockerfile:30]: We'll also need to `COPY` from `cinder-bundle` and then pass its CSV to the `csv-merger` below

Comment (dprince) [reply custom-bundle.Dockerfile:30]: I did this in a follow up. Thanks.

**Pull request merged**